commit 1f7097d09ce628878107ed30341cfc1eb3649a81
Author: Florian Weimer <fweimer@redhat.com>
Date:   Fri Jul 19 08:53:04 2019 +0200

    Linux: Update syscall-names.list to Linux 5.2
    
    This adds the system call names fsconfig, fsmount, fsopen, fspick,
    move_mount, open_tree.
    
    Tested with build-many-glibcs.py.

diff --git a/sysdeps/unix/sysv/linux/syscall-names.list b/sysdeps/unix/sysv/linux/syscall-names.list
index 95aa3ec7a5..21bf37c627 100644
--- a/sysdeps/unix/sysv/linux/syscall-names.list
+++ b/sysdeps/unix/sysv/linux/syscall-names.list
@@ -23,7 +23,7 @@
 # them.
 
 # The list of system calls is current as of Linux 5.1.
-kernel 5.1
+kernel 5.2
 
 FAST_atomic_update
 FAST_cmpxchg
@@ -124,7 +124,11 @@ fork
 fp_udfiex_crtl
 free_hugepages
 fremovexattr
+fsconfig
 fsetxattr
+fsmount
+fsopen
+fspick
 fstat
 fstat64
 fstatat64
@@ -248,6 +252,7 @@ mmap
 mmap2
 modify_ldt
 mount
+move_mount
 move_pages
 mprotect
 mpx
@@ -285,6 +290,7 @@ oldumount
 olduname
 open
 open_by_handle_at
+open_tree
 openat
 osf_adjtime
 osf_afs_syscall
